### PR TITLE
(doc) Edit SIGHUP predocs.

### PR DESCRIPTION
**Pull request by gguillotte**
Edits to the SIGHUP predocs by @cprice404. Resolves SERVER-1084 and SERVER-1192.

### CODE REVIEW

**Comment by fpringvaldsen** [comment]
:+1:

File: documentation/restarting.markdown [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+## Changes applied after a restart or SIGHUP
+
+### Change Puppet Server's configuration in `conf.d`
+
+To have Puppet Server apply changes to the [configuration files](./configuration.html) in its `conf.d` directory, you can either restart the process or send it a HUP signal.
+
+> **Note:** Changes to Puppet Server's [logging configuration in `logback.xml`][logback.xml] don't require a server reload or restart. Puppet Server recognizes and applies them automatically, though it can take a minute or so for this to happen.
+
+## Changes that require a full server restart
```
Comment (haus) [inline documentation/restarting.markdown]: Should we call out `bootstrap.cfg` as needing a full restart as well?

Comment (erikPrime) [inline documentation/restarting.markdown]: Yes: I threw a ticket somewhere specifically about that.

**Comment by rlinehan** [comment]
Feedback regarding bootstrap.cfg has been addressed, looks good to me. :+1:

**Pull request merged**